open_instruct/grpo_fast.py

```diff
@@ -2444,25 +2444,16 @@ def create_model_and_optimizer(
 
 def create_generation_configs(args: Args):
     """Create generation configs for training and evaluation."""
-    generation_config = vllm.SamplingParams(
+    generation_config = vllm_utils.SamplingConfig(
         temperature=args.temperature,
-        top_p=args.vllm_top_p,  # prevent rare out-of-vocab tokens with qwen
+        top_p=args.vllm_top_p,
         max_tokens=args.response_length,
-        include_stop_str_in_output=True,
-        skip_special_tokens=False,
         n=args.num_samples_per_prompt_rollout,
         stop=args.stop_strings,
         seed=args.seed,
-        logprobs=1,  # Enable logprobs to compare with local calculations
-        # IMPORTANT: Set output_kind to FINAL_ONLY to ensure vLLM V1 properly handles n>1
-        # With the default CUMULATIVE mode, vLLM V1 returns separate outputs for each
-        # completion, making it difficult to aggregate them correctly. FINAL_ONLY mode
-        # ensures all n completions are returned together in a single output.
-        output_kind=vllm.sampling_params.RequestOutputKind.FINAL_ONLY,
+        logprobs=1,
     )
-    eval_generation_config = generation_config.clone()
-    eval_generation_config.temperature = 0.0
-    eval_generation_config.n = 1
+    eval_generation_config = dataclasses.replace(generation_config, temperature=0.0, n=1)
```

Comment:
I know this is mimicking old behaviour, but maybe eval temp should actually just match train temp... since in the thinking era temp 0 actually makes perf worse.